bedrock/settings/__init__.py

```diff
@@ -154,10 +154,7 @@
     _csp_connect_extra_for_dev = [
         "com-getpocket-prod1.mini.snplow.net",
     ]
-    _csp_font_src = [
-        "'self'",
-        "assets.getpocket.com",
```

Comment:
@janbrasna @stevejalim was `assets.getpocket.com` safe to delete? I still see references to it in `media/css/pocket/utils/_fonts.scss`?

Reply:
Ah, nevermind - I see we're using `*.getpocket.com` now 👍

Reply:
It's not exactly nice TBH, all the wildcards should be tightened when time comes for #11943 but that needs reworking the joining logic, and selecting the (sub)domains (or even paths not just hosts) for each policy individually. I believe the fonts will survive until then, or until some prankster XSSes a font from `addons.m.o` 🤷